python/mxnet/numpy/multiarray.py

```diff
@@ -92,6 +93,16 @@
 _NDARRAY_NO_ZERO_DIM_BOOL_ARRAY = -1
 _NDARRAY_ZERO_DIM_BOOL_ARRAY_FALSE = 0
 _NDARRAY_ZERO_DIM_BOOL_ARRAY_TRUE = 1
+_SIGNED_INT32_UPPER_LIMIT = (2**31 - 1)
```

Comment:
since you are importing ndarray.ndarray here: https://github.com/apache/incubator-mxnet/blob/master/python/mxnet/numpy/multiarray.py#L51  
`_SIGNED_INT32_UPPER_LIMIT` is already defined in ndarray.py file: https://github.com/apache/incubator-mxnet/blob/master/python/mxnet/ndarray/ndarray.py#L60 
and can be imported like this: https://github.com/apache/incubator-mxnet/blob/master/python/mxnet/symbol/symbol.py#L40
So this won't be necessary.

Reply:
we are deleting ndarray later so the copy is intentional.